app/master/build.py

```diff
@@ -228,6 +229,27 @@ def complete_subjob(self, subjob_id, payload=None):
             self.mark_failed('Error occurred while completing subjob {}.'.format(subjob_id))
             raise
 
+    def _get_atom_artifact_sys_path(self, subjob_id, atom_id):
+        return os.path.join(
+            self._build_results_dir(),
+            Subjob.ATOM_DIR_FORMAT.format(
+                subjob_id,
+                atom_id,
+            ),
+        )
+
+    def _parse_payload_for_atom_exit_code(self, subjob_id):
```

Comment:
I'm going to use this same method to set the syspath for the console_output_file in the Atom class.

Because of that, do you mind renaming this function to something slightly more generic, maybe just "_parse_paylod_for_atom"?